tests/components/iaqualink/test_init.py

```diff
@@ -346,7 +346,7 @@ async def test_entity_assumed_and_available(
     light = get_aqualink_device(
         system, name="aux_1", cls=IaquaLightSwitch, data={"state": "1"}
     )
-    devices = {d.name: d for d in [light]}
+    devices = {d.name: d for d in (light,)}
```

Comment:
```suggestion
    devices = {light.name: light}
```
i think this should be the same

Reply:
I agree, suggestion applied.